Maestro/subscriptions.json

```diff
@@ -66,11 +66,17 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 6226
     },
+    // A build definition that will trigger an official build of the dotnet-docker repo
+    "dotnet-docker-pipebuild": {
+      "vsoInstance": "devdiv.visualstudio.com",
+      "vsoProject": "DevDiv",
+      "buildDefinitionId": 6226
+    },
     // A build definition capable of running any .ps1 script in the WCF repo. By default, the master branch.
     "wcf-general": {
       "vsoInstance": "devdiv.visualstudio.com",
       "vsoProject": "DevDiv",
-      "buildDefinitionId": 4226
+      "buildDefinitionId": 6412
```

Comment:
This change seems wrong... 4226 is the WCF generalexecutor, 6412 is Core-Setup 2.0.0 pipebuild.